### ISSUE TITLE
Fonts not loading when app is opened on other units.

### ISSUE THREAD

**Issue by cymophic** [PMC-EST/hr_lms#9]
**TL;DR**
Tried using ‘Poppins’ but it reverts back to the default font ‘Helvetica Neue’ on other systems. Switched to ‘Avenir’, as it’s native to Mac, and the app's users will be on Mac.

**Background**
I tried using 'Poppins' as the default font for the whole app. But when I asked Kenneth to open the app, the fonts are substituted back to 'Helvetica Neue' on his unit. After some research, I found out that there's no way of embedding fonts to an app in File Maker. Most of the system fonts are not good-looking, so this is a problem.

**Solution**
For now, I am using 'Avenir' font family, a decent system font close to 'Poppins' available natively on Mac. This would be a problem for users on Windows OS, but I've heard from sir Nur that HR-LMS will be mainly used by Mac users, so this workaround is okay 'till then.
